src/Route.elm

```diff
@@ -170,14 +174,14 @@ toUrlString route =
 
                 ByNamespace Relative fqn ->
                     if includeNamespacesSuffix then
-                        "latest" :: "-" :: NEL.toList (FQN.segments fqn)
+                        "latest" :: NEL.toList (FQN.segments fqn) ++ [ namespaceSuffix ]
 
                     else
                         "latest" :: NEL.toList (FQN.segments fqn)
 
                 ByNamespace (Absolute hash) fqn ->
                     if includeNamespacesSuffix then
-                        [ Hash.toUrlString hash, "namespaces" ] ++ NEL.toList (FQN.segments fqn) ++ [ "-" ]
+                        [ Hash.toUrlString hash, "namespaces" ] ++ NEL.toList (FQN.segments fqn) ++ [ namespaceSuffix ]
```

Comment:
These were just straight up bugs around the namespace suffix in the url. It was still "-" and it was even placed wrong the in the path...